### ISSUE TITLE
Site Accessibility #3

### ISSUE THREAD

**Issue by corysanders3** [corysanders3/travel-app#3]
The following must be met:
- [x] You must be able to tab through your app and use it without a mouse
- [x] Your app must still be usable when tested with a colorblind extension
- [x] You must score as close to 100% as possible with the Lighthouse and WAVE accessibility audits. Be prepared to explain any accessibility audits your application is failing.
- [x] Your HTML must be written semantically and should use ARIA attributes for any UI elements that are not understood by the screen reader (ONLY if needed / appropriate)
